pages.es/common/figlet.md

```diff
@@ -0,0 +1,20 @@
+# figlet
+
+> Genera encabezados usando caracteres ASCII desde la entrada del usuario.
+> Más información: <http://www.figlet.org/figlet-man.html>.
+
+- Genera el encabezado directamente introduciendo el texto:
+
+`figlet {{texto_de_entrada}}`
+
+- Usa un archivo de fuente personalizada:
+
+`figlet {{texto_de_entrada}} -f {{ruta/al/archivo_de_fuente}}`
+
+- Redirige la salida de un comando hacia figlet:
+
+`{{comando}} | figlet`
+
+- Muestra las fuentes disponibles para figlet:
+
+`showfigfonts {{texto_opcional_a_mostrar}}`
```

Comment:
I know the en has it however should it be here? (and on the other one)

Reply:
Yes, appears in English file, that's why included here.
I think it's related in some way with figlet. And thanks to be included here it's how I found that this command exits! :)
I'm a newcomer here, but I find it useful...